syft/src/main/java/org/openmined/syft/domain/JobRepository.kt

```diff
@@ -20,7 +20,7 @@ private const val TAG = "JobDownloader"
 
 @ExperimentalUnsignedTypes
 internal class JobRepository(
-    private val jobLocalDataSource: JobLocalDataSource,
+    val jobLocalDataSource: JobLocalDataSource,
     private val jobRemoteDataSource: JobRemoteDataSource
```

Comment:
why did you remove private?

Reply:
> why did you remove private?

Because I needed to access models path on SyftJob createDiff method